File: rest-service/src/main/java/mb/lib/db/model/ShortJobRow.java
package mb.lib.db.model;

import java.time.OffsetDateTime;

import mb.lib.model.JobStatus;

public interface ShortJobRow extends Row
{
  int queueID();

  OffsetDateTime createdOn();

  OffsetDateTime deleteOn();

  JobStatus status();

  String projectID();
}

File: rest-service/src/main/kotlin/mb/lib/db/model/ShortJobRow.kt
package mb.lib.db.model

import mb.lib.model.JobStatus
import java.time.OffsetDateTime

interface ShortJobRow: Row
{
  val queueID: Int

  val createdOn: OffsetDateTime

  val deleteOn: OffsetDateTime

  val status: JobStatus

  val projectID: String
}
